roles/splunk/tasks/configure_os.yml

```diff
@@ -12,7 +12,10 @@
 
 - name: Disable THP
   include_tasks: configure_thp.yml
-  when: "'full' in group_names"
+  when:
+    - ansible_service_mgr == "systemd"
+    - ansible_connection != "docker"
+    - "'full' in group_names"
```

Comment:
Are the double quotes around this necessary?

Reply:
Yes, it needs the double quotes